docs/data/charts/tooltip/tooltip.md

```diff
@@ -178,6 +178,8 @@ To apply the same style as we're trying to apply above, we need to use the `sx`
 
 {{"demo": "TooltipStyle.js"}}
 
+You can also disable the portal by setting `slotProps.tooltip.disablePortal` to `true`.
```

Comment:
Maybe we should mention on line 161 that we're using a portal, so that it's clear what disabling the portal will do. 

"You can also disable the portal" makes me question: which portal? And we don't see any mention of a portal before. 

We can maybe rewrite line 161 to be:

> By default, the tooltip is rendered as a child of the document's body using a portal, so using the chart's `sx` prop does not work.


And maybe on line 177, we can present both options at once:

> To apply the same style as we're trying to apply above, we need to:
> 1. Disable the portal by setting `slotProps.tooltip.disablePortal` to `true`, or;
> 2. Use the `sx` property inside `slotProps.tooltip`.
>
> The next example showcases the second option:

What do you think?

Reply:
I complitely reorganised this docs page in a follow-up PR. We can move this discussion there